quint/src/quintIr.ts

```diff
@@ -118,11 +118,16 @@ export interface QuintApp extends WithId {
   args: QuintEx[],
 }
 
+export interface QuintLambdaParameter extends WithId {
+  /** The name of the formal parameter */
+  name: string,
+}
+
 export interface QuintLambda extends WithId {
   /** Expressions kind ('lambda' -- operator abstraction) */
   kind: 'lambda',
-  /** Identifiers for the formal parameters */
-  params: string[],
+  /** The formal parameters */
+  params: QuintLambdaParameter[],
```

Comment:
To be clear, i expect there is a good reason, but i'm curious what it is :)

Reply:
Yep, so it would be tricky since, in many times we analyze a `NameEx`, we look for something already defined, which isn't the case here (i.e. the type inferrer, when visiting a `NameEx`, will use the lookup table to obtain the original definition and get the type for that)

Reply:
I see, thanks.